examples/carousel.amp.html

```diff
@@ -98,7 +98,7 @@
 
   </amp-carousel>
 
-  <amp-carousel width=auto height=300 controls>
+  <amp-carousel width=auto height=300 data-next-button-aria-label="Go Forward" data-previous-button-aria-label="Go Back" controls>
```

Comment:
nit: Let's use more descriptive values here to set a good best practice precedence. Maybe 

`Go to next slide` and `Go to previous slide`